CSS/styles.css

```diff
@@ -249,10 +250,9 @@ form ul {
   cursor: pointer;
 }
 
-.btn:hover {
-  background: #6070ff;
+.btn:active {
+  background: #2230d2;
   box-shadow: 0 8px 16px rgba(64, 83, 252, 0.24);
-  border-radius: 8px;
   color: #fff;
 }
```

Comment:
[OPTIONAL] As the [last code reviewer suggested](https://github.com/belalgamal46/Setup-and-mobile-first/pull/4#discussion_r983773217) please make sure that you are adding interactions to the buttons according to Figma template provided. The hover effect means that when you put the cursor over the button it changes its color, this feature is not working. Kindly fix the hover effect on the rest of the buttons in your project. :pray: :pray: 
![image](https://user-images.githubusercontent.com/98363075/193171016-4362958b-f66c-43ec-87fb-40bdd9bba3c5.png)